helm/cluster-api-provider-azure/templates/crd-install/crd-psp.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy" }}
```

Comment:
This also needs wrapping in an if block: https://github.com/giantswarm/cluster-api-provider-azure-app/blob/938a3fa153d6bc9d47672473971ed66a0352d4df/helm/cluster-api-provider-azure/templates/crd-install/crd-rbac.yaml#L31-L38

Reply:
Fixed. PTAL